src/unshare.c

```diff
@@ -0,0 +1,28 @@
+
+#define EXTUNIX_WANT_UNSHARE
+#include "config.h"
+
+#if defined(EXTUNIX_HAVE_UNSHARE)
+
+static int umountflags_table[] = {
+  CLONE_FS,  CLONE_FILES, CLONE_NEWNS, CLONE_SYSVSEM, CLONE_NEWUTS,
```

Comment:
`CLONE_NEWPID` and `CLONE_NEWUSER` seem to be rather recent editions, please add the test in `discover.ml` to require thei presence